rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -215,14 +353,65 @@ def _download_kubeconfig(ocm_cmnd, cluster_id, my_path):
         return kubeconfig_path
 
 
-def _build_cluster(ocm_cmnd, rosa_cmnd, cluster_name_seed, must_gather_all, mgmt_cluster_name, provision_shard, wait_time, cluster_load, load_duration, job_iterations, worker_nodes, my_path, my_uuid, my_inc, es, es_url, index, index_retry, all_clusters_installed):
+def _download_cluster_admin_kubeconfig(rosa_cmnd, cluster_name, my_path):
+    return_data = {}
+    logging.info('Creating cluster-admin user on cluster %s (5 retries allowed)' % cluster_name)
+    kubeconfig_cmd = [rosa_cmnd,  "create", "admin", "-c", cluster_name, "-o", "json"]
+    logging.debug(kubeconfig_cmd)
+    for trying in range(1, 11):
+        process = subprocess.Popen(kubeconfig_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=my_path, universal_newlines=True)
+        oc_login_start_time = int(time.time())
```

Comment:
We should have 2 times here IMHO. The initial time it takes for us to get a 0 rc from the create admin command and then one for how long after that it takes to get the kubeconfig. Right now it looks like we are only doing the second one.

Reply:
ok, collected both times now as:

metadata['cluster-admin-create'] = time until rosa create admin runs successfully (on all tests, its on the first try)
metadata['cluster-admin-login'] = time since rosa create admin runs sucessfully until we can login on the cluster